.yarnrc

```diff
@@ -0,0 +1,3 @@
+--install.modules-folder "./public/lib"
```

Comment:
@kaunair Why not install dependencies in default `node_modules/` folder?

Reply:
Oh I was just following the convention we followed for plots2
Changing it shouldn't be an issue, I presume. All you'll have to do is change the path rails loads assets from.

Reply:
Hmm :thinking:, It was just curiosity, I'll keep this. Only if @jywarren or @icarito wants me to change it.

Reply:
Yes, basically we do this for 2 reasons - one, it's then in a place where the Rails asset pipeline finds it, but more importantly it allows some assets to be loaded directly from a world-readable path instead of passed through the Rails application. (specifically image assets that are part of some NPM libraries). Thank you!!!